media-gfx/gimp/gimp-2.10.14.ebuild

```diff
@@ -0,0 +1,230 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+GNOME2_EAUTORECONF=yes
+
+inherit eapi7-ver virtualx autotools gnome2 python-single-r1
+
+DESCRIPTION="GNU Image Manipulation Program"
+HOMEPAGE="https://www.gimp.org/"
+SRC_URI="mirror://gimp/v$(ver_cut 1-2)/${P}.tar.bz2"
+LICENSE="GPL-3 LGPL-3"
+SLOT="2"
+KEYWORDS="~amd64"
```

Comment:
I think the necessary dependencies have been `~x86` keyworded meanwhile.

Reply:
It's related to https://bugs.gentoo.org/698696  
`media-libs/gegl-0.4.18` droped this keyword due to `media-libs/libnsgif`.
So I'm waiting when gentoo x86 arch team re-keyword them.
Actually both of them successfully built in x86 chroot env and test were passed for gegl.

Reply:
but `eshowkw` tells me both packages are already ~x86 keyworded again.

Reply:
Yes, since 14 nov. I was waiting notification in bugzilla and missed to recheck commits of these packages. Thank you for good news.

Reply:
Done. `x86` keyword is restored.